### ISSUE TITLE
Expose HSM status as an input select

### ISSUE THREAD

**Issue by amosyuen** [jason0x43/hacs-hubitat#151]
Currently it uses a sensor to read the state and a service to set it. But I think it would be easier to use if you use an input select. It also matches how the hub mode is exposed.

**Comment by jason0x43** [comment]
I agree, it would be a bit easier to use. It hasn't been implemented yet because HSM is a bit more complex than modes, but it's certainly doable.

Notes: The set of possible modes is listed in the [docs](https://docs.hubitat.com/index.php?title=Hubitat_Safety_Monitor), although I'm not sure I entirely trust that without more testing. The status values and values used to change the status aren't symmetric ("armedAway" vs "armed away"), and I think some potentially status values ("disarmed") have no equivalent status values (at least, there's no equivalent in the official list of possible states).
